bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -99,7 +99,7 @@ PreProcessor::PreProcessor(shared_ptr<MsgsCommunicator> &msgsCommunicator,
       requestsHandler_(requestsHandler),
       myReplica_(myReplica),
       myReplicaId_(myReplica.getReplicaConfig().replicaId),
-      maxPreExecResultSize_(myReplica.getReplicaConfig().maxReplyMessageSize - sizeof(ClientReplyMsgHeader)),
+      maxPreExecResultSize_(myReplica.getReplicaConfig().maxExternalMessageSize - sizeof(ClientReplyMsgHeader)),
```

Comment:
It should be just 'myReplica.getReplicaConfig().maxExternalMessageSize'. It does not have any connection to the reply msg.

Reply:
Fixed